taskcluster/client.py

```diff
@@ -603,5 +295,12 @@ def createTemporaryCredentials(clientId, accessToken, start, expiry, scopes, nam
 __all__ = ['createTemporaryCredentials', 'config']
 # This has to be done after the Client class is declared
 for key, value in API_CONFIG.items():
-    globals()[key] = createApiClient(key, value)
+    # import buildtime generated code if it exists; otherwise use runtime generated code
+    try:
+        module = __import__('taskcluster.sync.{key}'.format(key=key), globals(),
+                            locals(), [key], 0)
+        globals()[key] = getattr(module, key)
+    except (ImportError, AttributeError):
+        print("{} not in taskcluster.sync!!!".format(key))
```

Comment:
This is planning for a case that I don't think is likely.  If a service is in the Apis.json file, it will be generated in taskcluster.sync.  I'd rather just export the `createApiClient` method and force the export to use the taskcluster.sync methods.